distribution/docs/src/main/resources/content/_coreConcepts/catalog-framework-intro.adoc

```diff
@@ -9,7 +11,7 @@ The Catalog Framework wires all the Catalog components together.
 
 It is responsible for routing Catalog requests and responses to the appropriate source, destination, federated system, etc. 
 
-<<_endpoints,Endpoints>> send Catalog requests to the Catalog Framework.
-The Catalog Framework then invokes <<_types_of_plugins,Catalog Plugins>>, <<_transformers,Transformers>>, and <<_resources,Resource Components>> as needed before sending requests to the intended destination, such as one or more <<_introduction_to_federation_and_sources,Sources>>. 
+<<{integrating-prefix}endpoints,Endpoints>> send Catalog requests to the Catalog Framework.
```

Comment:
these `-prefix` properties will resolved to the document being referenced. In the `documentation.html` compendium, it will resolve to an internal reference.